eks/vpc.tf

```diff
@@ -32,8 +32,10 @@ module "vpc" {
   enable_dns_hostnames = true
   enable_dns_support   = true
 
-  enable_nat_gateway           = true
-  create_database_subnet_group = true
+  enable_nat_gateway = true
+
+  create_database_subnet_group = var.create_database_subnet_group
+  database_subnet_group_name   = var.database_subnet_group_name
```

Comment:
Description says `Default will be set to vpc name if not provided` - just checking this is true? (I assume if we don't implement this, then this must be how the underlying module behaves?)

Reply:
yep and that is what underlying module behaves :)